README.md

```diff
@@ -1,41 +1,8 @@
-<a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
+<a id="readme-top"></a>
```

Comment:
# Frontend comments for review #2 

- [ ] When I click any coach to reserve, I get this screen:

![image](https://user-images.githubusercontent.com/57421823/224164979-e1476d71-6c66-4b06-b814-92aca1fadee8.png)

It seems to be trying to get the coaches locally (as opposed to the deployed api) according to coach.js:

![image](https://user-images.githubusercontent.com/57421823/224165139-66ca7680-eb89-4a50-8b4d-ba188f0d8a22.png)

Also, that URL tries to get the API data directly to the browser in Rails, instead of consuming it with the frontend.

However, there is no route for that, as the error implies.  If you are trying to get the coaches and the whole project from the deployed API, please correct the URL.

- [ ] If I click on "add coach", I get this screen:

![image](https://user-images.githubusercontent.com/57421823/224167158-af6ca407-4621-4a73-98e4-ca9955c278b6.png)

Kindly, correct this bug so users have the ability to add coaches. 👍 

- [ ] In the mobile view, some elements get contracted and/or overflow:

![image](https://user-images.githubusercontent.com/57421823/224167602-0bb1942b-f7fc-40b0-a595-83a048a00850.png)

Kindly, add some styling so the elements are presented on the screen, without having to horizontally scroll.

- [ ] Since the last review, no changes were made to the frontend repo:

- App function is still not ES6.
- License still does not work on readme.
- Faq section on readme is still present and empty.
- The note placeholder in the license section is still present.